cmd/api/main_test.go

```
@@ -0,0 +1,51 @@
+func TestMain(m *testing.M) {
+	// テスト前の準備
+	setup()
+
+	// テストを実行
+	m.Run()
+
+	// テスト後のクリーンアップ
+	teardown()
+	// os.Exit(code)の代わりに、codeを返す
+	// テスト結果はm.Run()の戻り値として取得できるため、明示的にExitする必要はない
+}
```

golangci-lint (1.64.8)

block should not end with a whitespace (or comment)

(wsl)